canonical-web-and-design/ubuntu.com
# Fix responsiveness on /security/cve

## Done

- Update styles to fix responsiveness on mobile.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Be sure to test on mobile, tablet and desktop screen sizes
- You need to setup your local database with the cve data to see the table (Refer to [this](https://gist.github.com/goulinkh/3b4d6228de316d3a2cb2bf9450a97290) and let me know if you need anything about how to set up)
- Please check the responsiveness and check if everything works properly on all screens
## Issue / Card

Fixes #9325 

## Screenshots
Large screen :
![image](https://user-images.githubusercontent.com/57550290/144392153-0d34d0cf-1884-40a5-b0ee-e9e7d476659c.png)
Medium :
![image](https://user-images.githubusercontent.com/57550290/144390488-8d925670-093f-42a3-9e19-b8a17a905738.png)

Small: 
![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/57550290/144390995-21c6d46f-6a32-4e19-bac9-6075404e2693.gif)



## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
